docker-forward

```diff
@@ -267,7 +267,7 @@ def timeout(seconds=10, error_message=os.strerror(ETIME)):
 
     return decorator
 
-@timeout(30, "Are your containers running?")
+@timeout(5, "Are your containers running?")
```

Comment:
I believe this was set to 30 to allow time for the tunnels to be set up as the first time you run docker-forward it takes a bit.  Is 5 seconds enough time for all the tunnels to be created?

Reply:
Ya, 5 seconds isn't quite long enough for some cases.  I am thinking I'll just close this PR.